var/spack/repos/builtin/packages/libxsmm/package.py

```diff
@@ -14,10 +14,11 @@ class Libxsmm(MakefilePackage):
     and deep learning primitives."""
 
     homepage = 'https://github.com/hfp/libxsmm'
-    url      = 'https://github.com/hfp/libxsmm/archive/1.14.tar.gz'
+    url      = 'https://github.com/hfp/libxsmm/archive/1.15.tar.gz'
     git      = 'https://github.com/hfp/libxsmm.git'
 
     version('develop', branch='master')
```

Comment:
```suggestion
    version('master', branch='master')
```
While you're at it

Reply:
I remember, I was asked to name it "develop". Our development branch is the master branch. I am fine following your suggestion, just want to say: our development happens on master.

Reply:
Yep, that was because before #1983 was merged, Spack only considered `develop` as newer than stable releases. Now we consider `master` and other branch names as newer, so there's no need to name it differently.